recipes/autopxd2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "autopxd2" %}
+{% set version = "1.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autopxd2-{{ version }}.tar.gz
+  sha256: faa4b7eeb6b1d7217c46d3660640d4a72fb5c86d4559017c1f58a55d34332b2a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7,<3 | >=3.4
+  run:
+    - click
+    - pycparser
+    - python >=2.7,<3 | >=3.4
+    - six
+
+test:
+  imports:
+    - autopxd
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/gabrieldemarmiesse/python-autopxd2
+  summary: Automatically generate Cython pxd files from C headers
+  license: MIT
+  # Drop vendored license file once this upstream issue is addressed
+  # xref: https://github.com/gabrieldemarmiesse/python-autopxd2/issues/26
+  license_file: LICENSE
```

Comment:
Needed to vendor the license from upstream for now. Filed issue ( https://github.com/gabrieldemarmiesse/python-autopxd2/issues/26 ) about having it packaged